src/Components/Atoms/FARadio/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { memo } from "react";
+import { Radio, FormControlLabel } from "@mui/material";
+import { IFARadio } from "./FARadio.interface";
+
+export const FARadio = memo((props: IFARadio) => {
+  const { testID, label, ...materialUIProps } = props;
+  return (
+    <FormControlLabel
+      control={<Radio data-testid={testID} {...materialUIProps} />}
+      label={label || ""}
```

Comment:
if `label` is optional in `FARadio`, better to exclude `FormControlLabel` from this component.

Reply:
noted